fairchive-persistence/src/main/java/edu/harvard/iq/dataverse/persistence/dataset/DatasetField.java

```diff
@@ -308,6 +308,25 @@ public List<String> getValues() {
         return returnList;
     }
 
+    /**
+     * Returns field value (not formatted)
+     * or controlled vocab value list (not formatted)
+     */
+    public List<String> getValuesWithoutFormatting() {
```

Comment:
I'd optimize this a little

    public List<String> getValuesWithoutFormatting() {
        if (!getFieldValue().isEmpty()) {
            return Collections.singleonList(fieldValue);
        } else {
                    List<String> returnList = new ArrayList<>();
            for (ControlledVocabularyValue cvv : controlledVocabularyValues) {
                if (cvv != null && cvv.getLocaleStrValue() != null) {
                    returnList.add(cvv.getLocaleStrValue());
                }
            }
                   return returnList;
        }
    }